migrate-dao.js

```diff
@@ -397,6 +397,63 @@ async function migrateDAO ({ web3, spinner, confirm, opts, migrationParams, logT
     permissions.push('0x0000000A')
   }
 
+  for (const schemeName in migrationParams.CustomSchemes) {
+    let scheme = migrationParams.CustomSchemes[schemeName]
+    let { abi, bytecode } = require(`./custom-abis/${schemeName}.json`)
```

Comment:
Since migration will be imported as library. It would be better if the developer could provide path to the `custom-abis` folder

Reply:
Right, I added this option now.